### DISCUSSION TITLE
Integration with Alexa/Google home

### DISCUSSION THREAD

**Discussion by malhar-c** [HomeSpan/HomeSpan#324] [General]
Hi, 

I recently got into this, and have successfully made a prototype for my room (it can control couple relays for fan, lights, have a DHT sensor for temp and humidity and also can control my AC with IR blaster, here's the GitHub link if anyone is interested: https://github.com/malhar-c/Homespan-ESP32-SmartRoom), it works great with iOS, but I also want it to be controllable with an Android device, may be Alexa or google home integration will be great, can anyone suggest an option how to achieve this?

I installed homebridge on Windows to see if anything can be done, but cannot find any plugin for HomeSpan specifically, and moreover Homebridge alexa is no longer a free service, so not sure if it's an option anyway.

Any suggestion will be highly appreciated, I want to stay with this Library as this is great so far and have a lot of features which for me are yet to be discovered.

**Comment by HomeSpan** [comment]
I'm unfortunately not aware of any plug-ins that would allow HomeSpan devices to be controlled with Alexa.  However, HomeSpan devices behave the same as their commercial HomeKit counterparts, so any solution that enabled you to operate a commercial HomeKit device with Alexa would probably also work with HomeSpan devices.
